styles/globals.css

```diff
@@ -14,3 +14,15 @@ a {
 * {
   box-sizing: border-box;
 }
+
+:root {
+  --orange: #f37121;
+  --white: #ffffff;
+  --black: #1b1c1d;
+  --gray-1: #494c4e;
+}
+
+@font-face {
+  font-family: "Inter";
+  src: url(../assets/fonts/Inter.otf) format("opentype");
+}
```

Comment:
Yeah. Over here, I would add the different classes that show up on the Figma (like label large) etc. and the correct font size, font, and font-weight. Then, in any other file, we can just import the global.css file and use the headers.